tests/testsuite/bad_config.rs

```diff
@@ -983,6 +1071,42 @@ fn lib_crate_type2() {
         .run();
 }
 
+#[cargo_test(nightly, reason = "edition2024 is not stable")]
```

Comment:
Isn't this only because we report the field error before the "edition is unstable" error?  Removing this would make the test more brittle to change and is including more things in the test than are intended to be covered.

Reply:
Make sense. And believing it will be removed at the end of this year.